decidim-core/app/views/decidim/participatory_process_steps/_step.html.erb

```diff
@@ -0,0 +1,21 @@
+<% # TODO: Move to decorator? %>
+<% status_class = step.end_date > Time.zone.now  ? "timeline__item--inactive" : "" %>
+<% status_class = step.active? ? "timeline__item--current" : status_class %>
+
+<li class="<%= "timeline__item #{status_class}" %>">
+  <div class="timeline__phase">
+    <span class="timeline__phase__number"><%= step.position %></span>
+  </div>
+  <div class="timeline__info">
+    <div class="timeline__header">
+      <span class="timeline__date text-small">
+        <%= step.start_date.strftime("%d/%m/%Y") %> - <%= step.end_date.strftime("%d/%m/%Y") %>
```

Comment:
Careful with `strftime`, as different locales have different formats. Better use `localize`. in this case, looks like you should use `localize step.start_date, format: :short`. Also, take into account steps might now have neither start_date nor end_date, so you should put fallbacks for that.